src/base/RadioCheckboxBase.js

```diff
@@ -113,8 +113,7 @@ class RadioCheckboxBase extends React.PureComponent {
     const { aria, bkgSvgSprites, btnType, isDisabled, id, style, value } = this.props
     const { isSelected } = this.state
 
-    if (isDisabled) { SvgComponent = bkgSvgSprites.disabled }
```

Comment:
will obviously remove support for disabled sprite if we go with this, its currently the same as base for both checkbox and radio sans color